### PR TITLE
V1.0 model update

### PR DESCRIPTION
**Pull request by wkozmaNTIA**
Summary of updates:
- Fixed two bugs
- Updates the clutter model to align with the most recent revision, P.2108-1
- Adds unit tests
- Re-namespaces .NET wrapper to `ITS.ITU...`
- Changed .NET wrapper to use Enum as input

### CODE REVIEW

**Comment by wkozmaNTIA** [comment]
I forgot to mention that the data for the unit tests was generated by the independently developed implementation of the Rec (Excel xlsx file) that can be found here: https://www.itu.int/en/ITU-R/study-groups/rsg3/Pages/iono-tropo-spheric.aspx

File: src/InverseComplementaryCumulativeDistribution.cpp [hunk 1]
```diff
@@ -24,7 +24,7 @@ double InverseComplementaryCumulativeDistribution(double q)
     double D_3 = 0.001308;
```
Comment (blainNTIA) [inline src/InverseComplementaryCumulativeDistribution.cpp:24]: My really old copy of A&S and ITU-R P.1057 have C_0 set to 2.515517 (last digit is different).

**Review by blainNTIA — APPROVED** [review]
Looks good. There are a few typos but nothing that would warrant a re-review once fixed. So, I'm approving this but please review my comments.

File: src/InverseComplementaryCumulativeDistribution.cpp (hunk 1 above)
Comment (wkozmaNTIA) [reply src/InverseComplementaryCumulativeDistribution.cpp:24]: correct.  must have been a typo on my part.

**Pull request merged**